terraform/diff.go

```diff
@@ -279,6 +311,10 @@ func (d *ModuleDiff) String() string {
 	return buf.String()
 }
 
+func (d *ModuleDiff) GoString() string {
```

Comment:
Ah yeah, that is a real sadness. Perhaps we just use copystructure in GoString as well to get a non-pointer value. Realistically it should only happen for debugging... I'll remove it for now.